scripts/linters/pylint_extensions.py

```diff
@@ -2139,6 +2141,54 @@ def visit_classdef(self, node):
             self.add_message('no-dunder-metaclass', node=node)
 
 
+class StringConcatenationChecker(checkers.BaseChecker):
+    """Custom pylint checker which prohibits use of string concatenation and
+     promotes string interpolation instead.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'no-string-concatenation'
+    priority = -1
+    msgs = {
+        'C0035': (
+            'at %s, avoid using string concatenation and replace it with '
```

Comment:
Don't write the whole string the error message I think the line number and filename (printed by pylint itself) will be enough, you can just have "Please replace string concatenation with string interpolation".

Reply:
Great will implement it.